src/components/AddFile.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import style from "../scss/AddContent.scss";
+import Modal from "./Modal.jsx";
+
+const AddFile = props => {
+  let uploadFile = e => {
+    e.preventDefault();
+    const form = document.getElementById("fileForm");
+    const file = form[1].files[0];
+    const title = form[0].value;
+    let formData = new FormData();
+    formData.append("file", file);
+    formData.append("title", title);
+    formData.append(
+      "notegroup_id",
+      props.that.state.currentDirId[props.that.state.currentDepth]
```

Comment:
czy that jest instancją jakiegoś komponentu? 😨 
przekazuj explicit rzeczy których potrzebujesz jako props, a jeżeli uważasz że stworzyłoby to potrzebę przekazywania zbyt wielu propsów to czas na coś w stylu Reduxa albo Mobx ;) ewentualnie Context API